curricula/templates/curricula/partials/lesson_pills.html

```diff
@@ -1,18 +1,19 @@
+{% load i18n %}
 {% load mezzanine_tags %}
 <div class="resource-nav btn-toolbar" role="toolbar" aria-label="Lesson Resources">
     <div class="btn-group btn-group-xs" role="group" aria-label="PDF Buttons">
-        <a href="{{ unit.get_absolute_url }}" class="btn" role="button">Unit Overview</a>
-        <a href="{{ unit.get_pdf_url }}" class="btn" role="button">All Lessons PDF</a>
+        <a href="{{ unit.get_absolute_url }}" class="btn" role="button">{% trans 'Unit Overview' %}</a>
+        <a href="{{ unit.get_pdf_url }}" class="btn" role="button">{% trans 'All Lessons PDF' %}</a>
         {# Removing handouts PDF link from CSF until we can fix it #}
         {% if unit.has_resource_pdf %}
-        <a href="{{ unit.get_resources_pdf_url }}" class="btn" role="button">All Handouts PDF</a>
+        <a href="{{ unit.get_resources_pdf_url }}" class="btn" role="button">{% trans 'All Handouts PDF' %}</a>
         {% endif %}
         {% if changelog.count > 0 %}
 
             <div class="btn-group" role="group">
             <button type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
                     aria-expanded="false">
-                Last updated {{ changelog.first.revision.date_created|date:"n/j/y" }}
+                {% trans 'Last updated' %} {{ changelog.first.revision.date_created|date:"n/j/y" }}
```

Comment:
"Last updated" looks like it's used in 3 place: unit_pills.html, curriculum_pills.html, and here. I think this PR is good as is, but just noting that for the Jira item.